.github/workflows/learn-github-actions.yml

```diff
@@ -1,5 +1,14 @@
 name: learn-github-actions
-on: [push]
+on:
+  push:
+    branches:
+      - master
+      - newFeature
+
+  pull_request:
+    branches:
+      - newFeature
+
```

Comment:
In this case actions only run when push happens on the specidied branches, it will not run on new branches. Better just have `on: pull_request`